www/src/gatsby-plugin-theme-ui/components.js

```diff
@@ -6,13 +6,15 @@ import CloudCallout from "../components/shared/cloud-callout"
 import GuideList from "../components/guide-list.js"
 import Pullquote from "../components/shared/pullquote"
 import EggheadEmbed from "../components/shared/egghead-embed"
+import GraphqlEmbed from "../components/graphql-embed"
 import ComponentModel from "../components/layer-model/component-model"
 
 export default {
   CloudCallout,
   GuideList,
   Pullquote,
   EggheadEmbed,
+  GraphqlEmbed,
```

Comment:
Since this component is only used in a few docs, I think it's better to import it directly with:

```
import GraphqlEmbed from "@components/graphql-embed"
```

Please remove the changes to this file.